client/app/dedicatedCloud/components/activationStatus/constants.js

```diff
@@ -0,0 +1,33 @@
+export const COMPONENT_NAME = 'activationStatus';
```

Comment:
it would make more sense to have the component name in the .component file, it would be more conventional imho